src/Analysis/GWASWizard/shared/CustomDichotomousSelect.jsx

```diff
@@ -84,7 +84,7 @@ const CustomDichotomousSelect = ({
                 width: 300,
               }}
               actions={[
-                <DeleteOutlined onClick={(d) => handleCDRemove(d.uuid)} key='delete' />,
+                <DeleteOutlined onClick={() => handleCDRemove(cd.uuid)} key='delete' />,
```

Comment:
what is `cd`?

Reply:
an object in the [`selectedDichotomousCovariates`] array its mapping over

Reply:
i could rename

Reply:
it's not the name, I was just checking if it is not `undefined`